cloud/pkg/cloudstream/config/config.go

```diff
@@ -67,7 +65,7 @@ func InitConfigure(stream *v1alpha1.CloudStream) {
 			Config.Cert = cert
 			Config.Key = key
 		} else if !(cert == nil && key == nil) {
-			klog.Exit("Both of tunnelCert and key should be specified!")
+			panic("Both of tunnelCert and key should be specified!")
```

Comment:
Plz input an error value to panic(..) func

Reply:
Thanks, Fixed this